src/api/player_view_functions.clj

```diff
@@ -1,12 +1,13 @@
 (ns api.player-view-functions
   (:require [rules.victory-conditions :as victory]
+            [rules.abilities :as abilities]
             [configs.messages :as messages]))
 
-(defn ^:private partial-card
-  "Returns only certain keys of a card and change the :owner"
-  [card wanted-keys owner]
-  (assoc (select-keys card wanted-keys)
-         :owner owner))
+(defn ^:private modified-map
```

Comment:
I agree that former name might be not clear enough.
I changed it so it's more consistent with how it's used and is readable.
Also, this new function will be equally useful if new keys have to be added due to future rules or frontend needs.

Reply:
what about simply `(select-and-merge`? Hard to be more descriptive than that :upside_down_face: